### PR TITLE
made genai tutorials direct onnx tutorial links. + added goodnotes testimonial

### PR DESCRIPTION
**Pull request by MaanavD**
At some point will need to merge cards + testimonials objects to be one.

As always, preview at: https://maanavd.github.io/onnxruntime/

### CODE REVIEW

**Review by sophies927 — COMMENTED** [review]
The quotes for the Hugging Face and Goodnotes testimonials are now the same, so I think one of them is incorrect.

Also, it's unclear what the "Accelerated Inference API" is, so it may be worth linking if that's a publicly accessible product they work on (or removing the "...and accelerate private models..." part of the sentence.

File: src/routes/testimonials/+page.svelte [hunk 1]
```diff
@@ -75,103 +76,111 @@
...
 		},
 		{
-			title: 'Intelligenza Etica',
+			title: 'Goodnotes',
 			quote:
-				"We integrate AI models in various markets and regulated industries using many stacks and frameworks, merging R&D and Ethics. With ONNX Runtime, we provide maximum performance and flexibility to use the customers' preferred technology, from cloud to embedded systems.",
-			author: 'Mauro Bennici, AI Architect and AI Ethicist, Intelligenza Etica',
-			imgsrc: intelligenzaeticablogo,
-			imgalt: 'Intelligenza Etica logo'
+			"Thanks to ONNX Runtime Web, Goodnotes has seamlessly implemented Scribble to Erase. The first Goodnotes AI feature for Android, Windows, and Web	, delivering lightning-fast performance and an incredibly smooth user experience. It's a game-changer!",
```
Comment (sophies927) [inline src/routes/testimonials/+page.svelte:133]:
```suggestion
			"Thanks to ONNX Runtime Web, Goodnotes has seamlessly implemented Scribble to Erase. The first Goodnotes AI feature for Android, Windows, and Web, delivering lightning-fast performance and an incredibly smooth user experience. It's a game-changer!",
```

**Review by sophies927 — APPROVED** [review]
LGTM! Just made the tiniest syntax suggestion for formatting purposes.

**Comment by parinitarahi** [comment]
Approved

**Review by parinitarahi — APPROVED** [review]

**Pull request merged**